src/OrleansRuntime/Catalog/GrainCreator.cs

```diff
@@ -22,7 +25,16 @@ public GrainCreator(IGrainRuntime grainRuntime, IServiceProvider services = null
         {
             _grainRuntime = grainRuntime;
             _services = services;
```

Comment:
That can't change. I'm good with this for now, I think we'll need proper planning in this area.